en/snap/advanced-feature.md

```diff
@@ -551,30 +560,31 @@ curl -X POST \
 ```
 <!-- tabs:end -->
 
-Be sure to use the same `user_id` for that particular customer on future transactions. Their card will be previewed on Snap card payment page, on future transactions.
+Use the same `user_id` for that particular customer on future transactions. Their card will be previewed on Snap card payment page, on future transactions.
 
 ![enabled payment card](./../../asset/image/snap-adv-save-card-preview.jpg)
 
-To better understand the use cases, you an also further refer to [this article](https://support.midtrans.com/hc/en-us/articles/360002419153-One-Click-Two-Clicks-and-Recurring-Transaction).
+For more use cases, refer to [One Click, Two Click, and Recurring Transaction](https://support.midtrans.com/hc/en-us/articles/360002419153-One-Click-Two-Clicks-and-Recurring-Transaction).
 
 ### Recurring / Subscription Card Transaction
-Snap can be utilized **to initialize** subscription or recurring payment flow. Note that:
+Snap can be utilized to initialize subscription or recurring payment flow. Note that:
```

Comment:
`to initialize` is intentionally bolded, because of this reason explained: https://github.com/Midtrans/technical-documentation-site/pull/32/files#r534014840